docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md

```diff
@@ -0,0 +1,32 @@
+# Aliquot Ensemble Somatic Variant Merging and Masking
+
+## Description ##
+
+Aliquot Ensemble Somatic Variant Merging and Masking is a somatic variant calling pipeline used in GDC whole exome sequencing (WXS) and targeted sequencing harmonization.
+
+## Overview ##
+
+The Aliquot Ensemble Somatic Variant Merging and Masking workflow in the GDC harmonizes whole exome sequencing (WXS) and targeted sequencing data by identifying and merging somatic variants. This workflow consists of Somatic Aggregation and Masked Somatic Aggregation, which respectively generate comprehensive mutation annotation files (MAFs) and produce filtered, public-access MAFs by removing potentially identifiable germline mutation information.
```

Comment:
Change "by identifying and merging somatic variants." to "by merging multiple MAFs that were each generated by one variant caller."

Reply:
changed